test/model.test.js

```diff
@@ -5732,6 +5732,51 @@ describe('Model', function() {
 
   });
 
+  it('bulkwrite should not change updatedAt on subdocs when timestamps set to false (gh-13611)', async function() {
+
+    const postSchema = new Schema({
+      title: String,
+      category: String,
+      isDeleted: Boolean
+    }, { timestamps: true });
+
+    const userSchema = new Schema({
+      name: String,
+      isDeleted: Boolean,
+      posts: { type: [postSchema] }
+    }, { timestamps: true });
+
+    const User = db.model('gh13611User', userSchema);
+
+    const entry = await User.create({
+      name: 'Test Testerson',
+      posts: [{ title: 'title a', category: 'a', isDeleted: false }, { title: 'title b', category: 'b', isDeleted: false }],
+      isDeleted: false
+    });
+    const initialTime = entry.posts[0].updatedAt;
+
+    await User.bulkWrite([{
```

Comment:
Can we sleep for 5 milliseconds here to guarantee that the `bulkWrite` wouldn't be executed at the same exact millisecond and prevent false-positives in the future?

Reply:
:+1: for this